packages/discord-request/jest.config.cjs

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Imagine using `module.exports` in 2021

Reply:
Really wish ESM had more hooks so I didn't have to transpile just to test LOL